src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -76,11 +84,13 @@ public function __construct(
      */
     public function thereIsUserIdentifiedBy($email, $password)
     {
-        $customer = $this->createCustomer();
+        $customer = $this->createCustomer($email);
+        $this->customerRepository->add($customer);
```

Comment:
No need to add customer and then user. Flushing user should persist also customer.